lib/web_ui/lib/src/engine/surface/scene_builder.dart

```diff
@@ -541,15 +546,19 @@ class SurfaceSceneBuilder implements ui.SceneBuilder {
   /// cannot be used further.
   @override
   SurfaceScene build() {
-    _persistedScene.preroll();
-    if (_lastFrameScene == null) {
-      _persistedScene.build();
-    } else {
-      _persistedScene.update(_lastFrameScene);
-    }
-    commitScene(_persistedScene);
-    _lastFrameScene = _persistedScene;
-    return SurfaceScene(_persistedScene.rootElement);
+    timeAction<void>('sceneBuilderPrerollFrame', () {
```

Comment:
treeshake from release builds ?

Reply:
I couldn't find a way to tree-shake `timeAction` entirely without making the cost super-ugly. Open to ideas, but currently decided that it wasn't worth it. In release mode `timeAction` becomes:

```
R timeAction<R>(String label, Action<R> action) => action();
```

So the cost is one closure. Since `SceneBuilder.build` runs once per frame the difference is negligible.